pages/common/cat.md

```diff
@@ -9,16 +9,16 @@
 
 - Concatenate several files into an output file:
 
-`cat {{path/to/file1}} {{path/to/file2}} > {{path/to/output_file}}`
+`cat {{path/to/file1 path/to/file2 ...}} > {{path/to/output_file}}`
 
-- Append several files into an output file:
+- Append several files to an output file:
 
-`cat {{path/to/file1}} {{path/to/file2}} >> {{path/to/output_file}}`
+`cat {{path/to/file1 path/to/file2 ...}} >> {{path/to/output_file}}`
 
-- Number all output lines:
+- Copy the contents of a file into an output file without buffering:
 
-`cat -n {{path/to/file}}`
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
```

Comment:
Warning: [`-u` is ignored](https://manned.org/cat.1).
Note: current style guide [fallbacks](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax) to short options in this case but doesn't specify whether to keep not portable option or remove.

Reply:
My sorry I have some serious trouble parsing half of your comment.
I don't get what you mean by "not portable option[s]". Are you referring to `-u`? It is portable and this whole PR's reason to exist is that it is.
Some implementations might ignore it (every day I'm learning to hate GNU a bit more), but that doesn't matter as [many](https://manned.org/cat/df80928c) [do](https://man.openbsd.org/cat) [not](https://keith.github.io/xcode-man-pages/cat.1.html) (or at least they say they don't).

Reply:
> I'm sorry I have some serious trouble parsing half of your comment. I don't get what you mean by "not portable option[s]". Are you referring to `-u`? It is portable and this whole PR's reason to exist is that it is. Some implementations might ignore it (every day I'm learning to hate GNU a bit more), but that doesn't matter as [many](https://manned.org/cat/df80928c) [do](https://man.openbsd.org/cat) [not](https://keith.github.io/xcode-man-pages/cat.1.html) (or at least they say they don't).

Agreed, also it seems to be included in the recent POSIX specification [Ref](https://pubs.opengroup.org/onlinepubs/9699919799/toc.htm).